hw/dv/sv/usb20_agent/usb20_agent.core

```diff
@@ -15,6 +15,7 @@ filesets:
       - usb20_agent_cfg.sv: {is_include_file: true}
       - usb20_agent_cov.sv: {is_include_file: true}
       - usb20_item.sv: {is_include_file: true}
+      - usb20_driver.sv: {is_include_file: true}
```

Comment:
you need to add usb20_block_if.sv here too.

Reply:
Okay.  Sorry i missed.